src/theme/mui.ts

```diff
@@ -407,7 +406,7 @@ const theme = createMuiTheme({
     MuiCheckbox: {
       colorSecondary: {
         '&$disabled': {
-          color: rgba(secondary, 0.5),
+          color: `rgba(${secondary}, 0.5)`,
```

Comment:
I'm pretty sure this won't work as `secondary` is a hexadecimal color.

Reply:
what we can do, it to change it to 8-Digit Hex Code https://caniuse.com/css-rrggbbaa

Reply:
it should be something like `${secondary}7f` https://stackoverflow.com/a/8254129/2524608